run/data/data.go

```diff
@@ -48,32 +48,72 @@ type Schema struct {
 
 // Table is the data about a DB Table.
 type Table struct {
-	Name          string             // the converted name of the table
-	DBName        string             // the original name of the table in the DB
-	Schema        *Schema            `yaml:"-" json:"-"` // the schema this table is in
-	Columns       Columns            // Database columns
-	ColumnsByName map[string]*Column `yaml:"-" json:"-"` // dbname to column
-	PrimaryKeys   Columns            // Primary Key Columns
+	Name                               string                      // the converted name of the table
+	DBName                             string                      // the original name of the table in the DB
+	Schema                             *Schema                     `yaml:"-" json:"-"` // the schema this table is in
+	Columns                            Columns                     // Database columns
+	ColumnsByName                      map[string]*Column          `yaml:"-" json:"-"` // dbname to column
+	ForeignColumnsByForeignKey         map[string][]*ForeignColumn // foreign key columns
+	PrimaryKeys                        Columns                     // Primary Key Columns
+	ForeignKeys                        []string                    // database names of this tables foreign keys
```

Comment:
I would make this a ForeignKeys type that is a []*ForeignKey and give it a .Names the way we do with Columns.